Exiled.Events/EventArgs/Scp106/StalkingEventArgs.cs

```diff
@@ -22,26 +24,31 @@ public class StalkingEventArgs : IPlayerEvent, IDeniableEvent
         /// Initializes a new instance of the <see cref="StalkingEventArgs"/> class.
         /// </summary>
         /// <param name="player"><inheritdoc cref="Player"/></param>
-        /// <param name="scp106StalkAbility"><inheritdoc cref="Scp106StalkAbility"/></param>
         /// <param name="isAllowed"><inheritdoc cref="IsAllowed"/></param>
-        public StalkingEventArgs(Player player, Scp106StalkAbility scp106StalkAbility, bool isAllowed = true)
+        public StalkingEventArgs(Player player, bool isAllowed = true)
         {
             Player = player;
-            Scp106StalkAbility = scp106StalkAbility;
+            Scp106 = player.Role.As<Scp106Role>();
             IsAllowed = isAllowed;
-            Vigor = scp106StalkAbility.Vigor.VigorAmount;
             MinimumVigor = Scp106StalkAbility.MinVigorToSubmerge;
         }
 
         /// <summary>
         /// Gets the <see cref="Scp106StalkAbility"/>.
         /// </summary>
-        public Scp106StalkAbility Scp106StalkAbility { get; }
+        [Obsolete("Use Scp106.StalkAbility instead of this")]
+        public Scp106StalkAbility Scp106StalkAbility => Scp106.StalkAbility;
 
         /// <summary>
         /// Gets or sets the current vigor when SCP-106 starts to stalk.
         /// </summary>
-        public float Vigor { get; set; }
+        public float Vigor
```

Comment:
Why don't put obsolete here?

Reply:
This PR refers to a breaking change.

Reply:
I'm about why not put obsolete exactly to property and not to getter and setter

Reply:
firstly is not a BreakingChange since people can still use it
secondly is because get and set are different 'method'